testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -450,7 +450,7 @@ QUnit.test('From renders editors with the right label, labelMode', function(asse
         const widgetLabelText = renderedWidget.option('label');
 
         assert.equal(widgetLabelMode, labelMode === 'outside' ? 'hidden' : labelMode);
-        assert.equal(widgetLabelText, labelMode === 'outside' ? '' : 'Name:');
+        assert.equal(widgetLabelText, 'Name:');
```

Comment:
This test expects dxTextEdit and it' better to explicitly specify it in the Form options:

        const form = $('#form').dxForm({
            labelMode,
            items: [ { editorType: "dxTextBox" } ]
        }).dxForm('instance');

Reply:
yep, fixed